src/program/loadtest/find-limit/find-limit.lua

```diff
@@ -89,6 +83,14 @@ local function parse_args(args)
                          ["retry-count"]="r", help="h", cpu=1,
                          exec="e"})
 
+   if #args == 2 then
+      -- Assume legacy mode
+      args = {
+         args[2], -- PCAP file
+         'NIC 0', 'NIC 0',
```

Comment:
Can this just be NIC?  I mean there's only one in this case, no need to differentiate.